### DISCUSSION TITLE
Classe et maîtrise

### DISCUSSION THREAD

**Discussion by ghost** [Crownicles/Crownicles#1974] [Nouvelles idées]
Si un jour on a des classes de tiers 6, je vois bien différente choses.
1- Mettre les t6 au level 100 et y'aurait jamais de t7 car 100 c'est une bonne symbolique.
2- Introduire la maîtrise de classe.

Je m'explique, maitrîser une classe permettrait d'avoir un bonus de stats sur celle-ci MAIS si l'on change de classe, on perdrait la maitrise.

Je vois la maîtrise un peu comme les slots d'inventaire. On pourrait payer (cher) une étoile de maitrise afin d'augmenter celle-ci et ça coûterait de plus en plus cher. Et je le vois en 3 tiers.
- Le maximum d'étoiles cumulables sur une classe serait de 5 étoiles.
- Il y aurait 3 étoiles différentes : bronze, argent et or.
- 5 étoiles de bronze = 1 d'argent et 5 d'argent = 1 d'or.
Donc quand on a 5 étoiles d'un même tiers, on peut evolve. (Sera automatiquement converti lors de l'achat de la 5e étoile d'un tier)

Donc le maximum de la maitrise d'une classe serait de 5 étoiles d'or.

**Comment by ghost** [comment]
Ou à la place de payer j'ai aussi penser gagner x combats pour gagner une maîtrise.

**Comment by Nargacaura** [comment]
À mon avis, ça serait une bonne idée d'avoir un palier final au niveau 100, mais je n'suis pas sûr si un nouveau palier sera ajouté.

Pour la maitrise... J'allais dire que ça ressemblait à la #1682, mais cette interprétation est meilleure que la mienne à mon goût xD

**Reply by Bombafox** [reply]
Dans l'idée des t6 au niv 100 c'est surtout de pouvoir utiliser nos arme à plein potentiel
Genre moi j'ai 650 d'attaque mais ça me sert à rien
Donc tout ce qui est item d'attaque ou popo sont inutiles
Donc ce serait cool de faire en sorte que par exemple si on a plus de 1k d'attaque ça change rien après
(oui je pense aussi au haut niv et surtout si c'est le dernier truc qui change)
Les stats sont peut être un peu abusé mais c'est pour l'idée
